src/js/extensions/auto-link.js

```diff
@@ -205,7 +205,7 @@ LINK_REGEXP_TEXT =
 
         createAutoLink: function (textNodes, href) {
             href = Util.ensureUrlHasProtocol(href);
-            var anchor = Util.createLink(this.document, textNodes, href),
+            var anchor = Util.createLink(this.document, textNodes, href, this.getEditorOption('targetBlank')),
```

Comment:
Yeah, it shouldn't be removed.
I think it could lead to a confusion since both method _seem_ to the same action and they have the same name